src/main/java/frc/robot/RobotContainer.java

```diff
@@ -78,20 +83,49 @@ public RobotContainer(RobotConfig rc) {
     defineSubsystems();
     defineButtons();
     configureButtonBindings();
-    defineAutonomousComponents();
     setDefaultDriveCommand();
-    initAutoShuffleboardCommands();
+    setupShuffleboard();
+
+    limelightSubsystem.off(); // turn the light off upon startup
   }
 
+  DoubleSupplier m_translationXSupplier = new DoubleSupplier() {
+    @Override
+    public double getAsDouble() {
+      var input = -modifyAxis(m_xspeedLimiter.calculate(joystick1.getY())) * Constants.TRAINING_WHEELS;
+      var speed = input * Constants.MAX_VELOCITY_METERS_PER_SECOND;
+      speed = highPassFilter(speed, Constants.MIN_VELOCITY_METERS_PER_SECOND);
+      return speed;
+    }
+  };
+
+  DoubleSupplier m_translationYSupplier = new DoubleSupplier() {
+    @Override
+    public double getAsDouble() {
+      var input = -modifyAxis(m_yspeedLimiter.calculate(joystick1.getX())) * Constants.TRAINING_WHEELS;
+      var speed = input * Constants.MAX_VELOCITY_METERS_PER_SECOND;
+      speed = highPassFilter(speed, Constants.MIN_VELOCITY_METERS_PER_SECOND);
+      return speed;
+    }
+  };
+
+  DoubleSupplier m_rotationSupplier = new DoubleSupplier() {
+    @Override
+    public double getAsDouble() {
+      var input = 0.0;
+      if (limelightSubsystem != null && limelightRotation) {
+        input = limelightSubsystem.rotation.getAsDouble() * 0.15;
+      } else {
+        input = (-modifyAxis(joystick2.getX())) * Constants.TRAINING_WHEELS;
+      }
+      var speed = input * Constants.MAX_ANGULAR_VELOCITY;
+      speed = highPassFilter(speed, Constants.MIN_ANGULAR_VELOCITY);
+      return speed;
+    }
+  };
```

Comment:
I'm okay with this for now, but I think, in the future, the translation/rotation suppliers should be treated as human input suppliers only (we already control the robot differently in auto).
Commands should be able to require a drivetrain and the default command should continue to work as expected after that command completes